qiime2_TagSequencingReport.Rmd

```diff
@@ -0,0 +1,383 @@
+<!--   Beginning of Preamble : Preamble seldom requires change                                        -->
+<!-- ================================================================================================ -->
+
+<!-- knitr setup -->
+```{r knitr setup, echo=FALSE}
+### ================
+###   knitr setup
+### ================
+library(knitr)
+opts_chunk$set(
+  tidy=FALSE,
+  cache=FALSE,
+  echo=FALSE,
+  warning=FALSE,
+  message=FALSE,
+  dpi=100,
+  fig.width=8,
+  fig.height=8,
+  fig.align = "center"
+  )
+```
+
+<!-- R packages -->
+```{r R packages, message=FALSE}
+### ================
+###   R packages
+### ================
+library(plyr)
+library(dplyr)
+library(qiimer)
+library(pander)
+library(ape)
+library(vegan)
+library(ggplot2)
+library(tidyr)
+library(broom)
+library(reshape2)
+library(kylemisc)
+library(gtools) 
+```
+
+<!-- resources -->
+```{r resources}
+### ================
+###   R resources
+### ================
+source("R_functions.R")
+```
+
+<!-- user defined functions -->
+```{r user defined functions}
+### ================
+###   User defined functions, if any
+### ================
+```
+
+<!-- ================================================================================================ -->
+<!--   End of Preamble                                                                                -->
+<!-- ================================================================================================ -->
+
+<!-- ================================================================================================ -->
+<!--   Beginning of Project Specific Items                                                            -->
+<!-- ================================================================================================ -->
+
+```{r}
+### ===========================
+###   define constants
+### ===========================
+
+### run date
+run_date <- "October 3rd, 2017"
+
+### investigator
+investigator <- "Mark Brown"
+
+### data files directory
+data_files_dir <- "data_files"
+
+### porject directory
+proj_dir <- "16s"
+
+### mapping file
+mapping_file_fp <- file.path(data_files_dir, "20171003_brown_mapping_file.txt")
+
+### per-sample read count 
+read_count_fp <- file.path(proj_dir, "table", "sample-frequency-detail.csv")
+
+### otu table
+feature_table_fp <- file.path(proj_dir, "table", "feature-table.tsv")
+
+### taxonomic assignment 
+taxo_assignment_fp <- file.path(proj_dir, "taxonomy", "taxonomy.tsv")
+
+### unweighted UniFrac
+uu_fp <- file.path(proj_dir, "core-metrics-results", "uu", "distance-matrix.tsv")
+
+### weighted UniFrac
+wu_fp <- file.path(proj_dir, "core-metrics-results", "wu", "distance-matrix.tsv")
+
+### num of observed otu
+notu_fp <- file.path(proj_dir, "core-metrics-results", "observed_otu", "alpha-diversity.tsv")
+
+### shannon
+shannon_fp <- file.path(proj_dir, "core-metrics-results", "shannon", "alpha-diversity.tsv")
+
+### minimum reads threshold
+min_reads <- 1000
```

Comment:
Recommend moving this up to be close to the other commonly modified variables.

Reply:
moved up